core/src/main/scala/besom/internal/BesomModule.scala

```diff
@@ -52,7 +52,8 @@ trait EffectBesomModule extends BesomSyntax:
         config         <- Config.forProject(runInfo.project)
         featureSupport <- FeatureSupport(monitor)
         _              <- logger.trace(s"Environment:\n${sys.env.toSeq.sortBy(_._1).map((k, v) => s"$k: $v").mkString("\n")}")
-        _              <- logger.debug(s"Resolved feature support, spawning context and executing user program.")
+        _              <- logger.debug(s"Resolved feature support: ${pprint(featureSupport)}")
```

Comment:
Is pprint() returning a String or Unit? I am not sure if it just prints to stdout or converts into pretty printed string?

Reply:
it returns a Str and a toString is called, we already use it this way in the code so I did the same for consistency

Reply:
Ok, my mistake then